Cargo.toml

```diff
@@ -26,7 +26,7 @@ jstz_api = { path = "crates/jstz_api" }
 jstz_kernel = { path = "crates/jstz_kernel" }
 jstz_proto = { path = "crates/jstz_proto" }
 jstz_crypto = { path = "crates/jstz_crypto" }
-jstz_node = { path = "crates/jstz_node" }
+jstz_node = { path = "crates/jstz_node"}
```

Comment:
Nit: Strange the formatter didn't catch this, but this diff is unnecessary. Recommend reverting this